mysql/mysqldump.go

```diff
@@ -321,6 +360,11 @@ func processAlterTable(conv *internal.Conv, stmt *ast.AlterTableStmt) {
 					ctable.ForeignKeys = append(ctable.ForeignKeys, constraint.fk)
 					conv.SrcSchema[tableName] = ctable
 				}
+				if constraint.isUniqueKey {
```

Comment:
I think we are missing test cases for when ALTER TABLE adds a uniqueness constraint.

Reply:
added tests for ALTER TABLE statement